modules/CMakeLists.txt

```
@@ -9,8 +9,16 @@ add_subdirectory(funnyPostures)
 option(CREATE_ICUBWRITER "Build iCubWriter" OFF)
 option(CREATE_FUNNYTHINGSAPP "Build funnyThingsApp" OFF)
 if(CREATE_ICUBWRITER)
-  add_subdirectory(iCubWriter)
+add_subdirectory(iCubWriter)
```

```suggestion
  add_subdirectory(iCubWriter)
```

Let's keep indentation.